openslides_backend/action/create_action_with_inferred_meeting.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict, Type
+
+from ..models.fields import BaseGenericRelationField, BaseRelationField
+from ..shared.exceptions import ActionException
+from ..shared.patterns import FullQualifiedId
+from .generics import CreateAction
+
+
+class CreateActionWithInferredMeetingMixin(CreateAction):
+    """
+    Mixin to autmatically set the meeting_id on create if it's not given in the payload.
+    If related_collection_for_meeting is not given, the relation_field_for_meeting must be
```

Comment:
This docstring does not fit: There is no property "related_collection_for_meeting". Please check.